config/yamlcfg/yaml2proto_test.go

```diff
@@ -169,6 +170,7 @@ test_groups:
 			expectedDashTab:   20,
 		},
 		{
+			// TODO: Do we want to prevent this case?
```

Comment:
Prevent as in "fail if there are defaults specified outside of default.yaml"?

I think that might be clearer than warning that they do nothing (current behavior). If this is done with a clear enough error ("Please delete defaults in *.yaml, they don't do anything"), I'd support it.

Consider changing this TODO to "Yes, let's prevent this actually" and implementing in a separate PR.

Reply:
Done